hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieParquetConfig.java

```diff
@@ -34,6 +37,8 @@
   private final Configuration hadoopConf;
   private final double compressionRatio;
   private final boolean dictionaryEnabled;
+  private final long minRowCountForSizeCheck = DEFAULT_MINIMUM_RECORD_COUNT_FOR_CHECK;
```

Comment:
@boneanxs since there's already Parquet config for it, let's just re-use that (i was advising against introducing new Hudi config dedicated for it, but we should actually reuse existing Parquet's one)